src/collections/blog/2020-09-25-announcing-meshmates/index.mdx

```diff
@@ -0,0 +1,153 @@
+    helping others contribute, something which I had never had the opportunity to
+    do before! After months of continuous effort, I am proud to write that I
+    recently joined as a maintainer on the project and was soon approached to be a
+    MeshMate, which I happily accepted. My learning graph has seen a soaring
+    positive tangent in the months I have been at Layer5 and I’m looking forward
+    to an even more educative and rewarding journey ahead 😊
+  </p>
+  <br />
+  <h4>
+    <a href="https://layer5.io/community/members/kelechi-nwachukwu">
+      Kelechi Precious Nwachukwu
+    </a>
+  </h4>
+  <img
+    src={ Kelechi }
+    class="image-right"
+  />
+  <p>
+    It was the middle of the COVID-19 pandemic and being unproductive wasn't
+    something I can stomach. Wanting to expand on my technical domain knowledge, I
+    stumbled across an open-source challenge for women in tech with lists of
+    projects to contribute to by the Women of Open Source Community Africa
+    (WOSCA). My eyes landed on Layer5 and I haven’t looked back since.
+  </p>
+  <p>
+    A quick look through the repositories left me quaking in my boots. To give you
+    some much needed context and to give some hope to all the newbies out there, I
+    recently transitioned to working in technical domains from accounting and
+    managing construction site projects. Forever getting stuck trying to get my
+    hands on new tech and trying to find a way into the bewildering and humongous
+    tech domains , I didn't even understand some of the jargon, and was too
+    ashamed to ask questions. Stumbling through my first contribution, I somehow
+    made it to the finish line and to my happiness, was soon assigned another
+    task. Instead of complaining, I set out to study up on the issue and took a
+    peek at the other assignees in the repository to ensure that I was not the
+    only one taking inordinate amounts of time on an issue. To my surprise, I was
+    not.
+  </p>
+  <p>
+    I got working on the assignment and soon encountered issues that would
+    previously have made me quit. Thankfully, I decided to take a different
+    approach this time. I shared my challenge with the Layer5 community, and was
+    amazed at the immediate and abundant response I received. Since then, I have
+    helped other contributors get through their challenges, and have received an
+    invitation to join the organization as a member, and then as a MeshMate 😎 I
+    have been wholly satisfied with the learning opportunities and the friendly
+    environment that the Layer5 community has to offer. I have learned that I may
+    not be perfect, but by making full use of the resources provided to me, having
+    a supportive community by my side and by being open to learning and showing
+    determined courage in the face of difficulties, I can achieve anything!
+  </p>
+  <h4>
+    <a href="https://layer5.io/community/members/ruth-ikegah"> Ruth Ikegah </a>
```

Comment:
Change the link to layer5-ng websites' member page

Reply:
like above